modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -12,19 +12,17 @@ Runtime templates related to Anypoint Partner Manager 2.5 are upgraded when:
 
 == Perform the Upgrade
 
-When new template versions are released, you see the *Upgrade deployments* button enabled in the top right corner of the *Message Flows* page. 
+When new template versions are released, you see the *Upgrade deployments* button enabled in the top right corner of the *Message Flows* page.
 
-To take advantage of new features, click the *Upgrade deployments* button. The process of updating all your message flows to use the latest template version begins. 
+To take advantage of new features, click the *Upgrade deployments* button. The process of updating all your message flows to use the latest template version begins.
 
-This process can take several minutes, depending on the number of message flow configurations and runtime applications your environment has. 
+This process can take several minutes, depending on the number of message flow configurations and runtime applications in your environment.
```

Comment:
To take advantage of new features, click the *Upgrade deployments* button. The process of updating all your message flows can take several minutes, depending on the number of message flow configurations and runtime applications in your environment.